go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/aws/aws-sdk-go v1.29.29
 	github.com/davecgh/go-spew v1.1.1
 	github.com/google/go-cmp v0.4.0
+	github.com/pkg/errors v0.9.1
```

Comment:
This package shouldn't be used since we replace it with Go 1.13 error wrapping. Please run `go mod tidy`.